.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.12'
+          python-version: '3.13'
```

Comment:
The latest ansible-lint version somehow requires 3.13 specifically and won't work otherwise.